tests/baselines/reference/strictModeInConstructor.js

```diff
@@ -102,10 +103,10 @@ var C = /** @class */ (function (_super) {
 var D = /** @class */ (function (_super) {
     __extends(D, _super);
     function D() {
-        var _this = this;
+        var x = 1; // No error
+        var y = _this.s; // Error

```

Comment:
This is interesting. Because there is no longer a `var _this = this;` added to the beginning of the constructor, a reference to `_this` before `super()` in a derived constructor will now result in a `ReferenceError: _this is not defined`.

```ts
constructor() {
    this.weirdError(); // ReferenceError: _this is not defined
    super();
}
```

I'll look into fixing..